- Same change that is proposed here: https://github.com/USCiLab/cereal/issues/668
- Otherwise sandbox is always built.

--- CMakeLists.txt.orig	2020-11-17 20:50:06 UTC
+++ CMakeLists.txt
@@ -115,6 +115,8 @@ if(BUILD_TESTS)
     add_subdirectory(unittests)
 endif()
 
-add_subdirectory(sandbox)
+if(BUILD_SANDBOX)
+    add_subdirectory(sandbox)
+endif()
 
 add_subdirectory(doc)
